script/build.js

```diff
@@ -184,22 +184,15 @@ if (options.watch) {
     allowRedirects: true,  // Don't require trailing slash for index.html links.
     warn: process.env.NODE_ENV !== 'production',
     allowRegex: new RegExp(
-        [ '/disability-benefits/',
-          '/disability-benefits/apply-for-benefits/',
-          '/disability-benefits/learn/',
-          '/disability-benefits/learn/eligibility/.*',
+        [ '/disability-benefits/apply-for-benefits',
```

Comment:
I don't understand why this one is needed, but when I remove it I get:
```
Warning: Link is broken: href: "/disability-benefits/apply-for-benefits/active-duty/", text: "
              How do I file while on active duty?
            ", in file: disability-benefits/apply-for-benefits/help/index.html
```
Any thoughts @awong-dev or @bshyong ?